lfucg-exactions/server/plats/serializers.py

```diff
@@ -114,10 +136,25 @@ class Meta:
             'cleaned_acres',
         )
 
+class SubdivisionField(serializers.Field):
+    def to_internal_value(self, data):
+        try:
+            return Subdivision.objects.get(id=data)
+        except:
+            None
+
+    def to_representation(self, obj):
+        try:
+            subdivision = SubdivisionSerializer(obj).data
```

Comment:
since you already wrapped this in a try catch, use return SubdivisionSerializer(obj).data['id']